packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -12,7 +12,7 @@
 -
 -  # -- [Image pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images) for updating already existing images on a node.
 +  repository: rancher/mirrored-kube-logging-logging-operator
-+  tag: 4.8.0
++  tag: 4.9.1
```

Comment:
I think similar to comment feedback on the other PR for logging in 2.10 - we want the same consistency in logic for chart <-> image version resolution.

Reply:
Fixed as agreed on dev-v2.10 PR